scripts/scale/scale_utils.py

```diff
@@ -0,0 +1,31 @@
+import os
+from typing import Dict
+from scaleapi import ScaleClient
+
+
+def get_credentials(path: str) -> Dict[str, str]:
+    # Reads the credentials from the given path
+    with open(path, "r") as f:
+        # Read line by line, replaces the spaces, splits on the first ":"
+        # The first part is the key, the second part contians the value in between quotes
+        credentials = {}
+        for line in f.readlines():
+            elt = line.replace(" ", "").replace("\n", "").split(":")
+            if len(elt) == 2:
+                credentials[elt[0]] = elt[1].split('"')[1]
+        return credentials
+
+
+def get_scale_client() -> ScaleClient:
+    relative_credentials_path = "../../prod_env/credentials.conf"
```

Comment:
I'll allow this for now but this should:

1. be a flag
2. be `prod_env/credentials.conf` if we run from the Git root, because paths are relative to the working directory, not the Python module